### DISCUSSION TITLE
Mode display

### DISCUSSION THREAD

**Discussion by thomashirtz** [JetBrains/ideavim#664] [General]
Would it be possible to add an option to add which mode you are in at the bottom, similar to the original VIM interface ?

Thanks !

**Comment by citizenmatt** [comment]
The status bar should already 
 show insert, visual, select, etc.  Do you have any suggestions of what you'd like to see?

**Comment by ljchen1** [comment]
Do you use the new UI in IDEA?
Might be related to this issue: https://youtrack.jetbrains.com/issue/VIM-2889/ (IdeaVim message not shown if new UI enabled and navigation bar is shown in status bar)
Try the workaround here for it: https://youtrack.jetbrains.com/issue/VIM-2889/IdeaVim-message-not-shown-if-new-UI-enabled-and-navigation-bar-is-shown-in-status-bar#focus=Comments-27-6959356.0-0
